compiler/src/transform/filter-by-availability.ts

```diff
@@ -66,9 +66,9 @@ function filterModel(inputModel: Model, stack, serverless: boolean): Model {
         output.types.push(typeDef)
 
         // store the type infos to prevent doubles & recursive calls
-        seen.get(typeName.namespace) ?
-          seen.get(typeName.namespace)?.push(typeName.name) :
-          seen.set(typeName.namespace, [typeName.name])
+        ;(seen.get(typeName.namespace) != null)
+          ? seen.get(typeName.namespace)?.push(typeName.name)
+          : seen.set(typeName.namespace, [typeName.name])
```

Comment:
semi-nit: if you have to use a leading `;` to make a ternary readable, maybe switch to an `if`.

Also, change `!= null` to `!== null`, assuming you want a strict null check. If you use `!=`, some values like `undefined` will also compare as equal. Isn't JavaScript loose equality wonderful? :upside_down_face: 

```suggestion
        if (seen.get(typeName.namespace) !== null) {
          seen.get(typeName.namespace)?.push(typeName.name)
        } else {
          seen.set(typeName.namespace, [typeName.name])
        }
```